Makefile

```diff
@@ -50,11 +50,11 @@ buf-lint:  ## run linting
 
 .PHONY: darwin
 darwin: ## complie for darwin
-	GOOS=darwin ${BUILD_ARGS} -o bin/${BINARY}-darwin-amd64 main.go
+	GOOS=darwin ${BUILD_ARGS} -o bin/${BINARY}-darwin-amd64 cmd/pbnj/*.go
```

Comment:
You don't need the `*.go`, `cmd/pbnj` is good enough

Reply:
Sweet, good to know. Thanks!